packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -137,31 +126,28 @@ export async function unlinkPatientFromOrganization({
   log(`Completed unlinking patient from organization`);
 }
 
-function findCwLinkWithOid(cwPatientData: CwData | undefined, oid: string): CwLink | undefined {
+function findCwLinkWithOid(cwPatientData: CwData | undefined, oid: string): CwLinkV2 | undefined {
   if (!cwPatientData) return undefined;
 
   const cwLinks = cwPatientData.links;
+  if (cwLinks.some(isCwLinkV1)) {
+    throw new BadRequestError("Patient contains CW v1 links. Rerun PD to continue.", undefined, {
+      oid,
+      cwLink: cwLinks.map(link => link.toString()).join(", "),
```

Comment:
how come we wanted to add all links to the log as opposed to just the cw v1 links?